src/domain/entities/Permission.ts

```diff
@@ -10,4 +11,14 @@ export class Permission extends Struct<PermissionAttrs>() {
     static buildWithOutAccess(): Permission {
         return Permission.create({ read: false, write: false });
     }
+
+    static setDefaultPermissionsForGroups(isAdmin: boolean): Permissions {
+        const adminDataPermission = Permission.create({ read: true, write: true });
```

Comment:
declarative: `adminDataPermission` -> `readWritePermission` (what things are, not how they are used / assigned)